src/Groups/directproducts.jl

```diff
@@ -141,6 +168,22 @@ end
     as_perm_group(G::DirectProductGroup)
 
 If `G` is direct product of permutations groups, return `G` as permutation group.
+
+# Examples
+```jldoctest
+julia> H = symmetric_group(3)
+Sym( [ 1 .. 3 ] )
+
+julia> K = symmetric_group(2)
+Sym( [ 1 .. 2 ] )
+
+julia> G = direct_product(H,K)
+DirectProduct of
+
+
+julia> as_perm_group(G)
```

Comment:
Another thing you could work on (maybe not in this PR but a follow-up): replace `as_perm_group` and `as_polycyclic_group` by methods for `PermGroup` and `PcGroup`. That would fit better with how we do this elsewhere! Also, a similar handling but for `FpGroup` seems to be missing.